build_scripts/Debian+Ubuntu/Linux_InstallGuide.md

```diff
@@ -98,7 +96,7 @@ The executable produced will be:
 
 The executable produced will be:  
 ```bash
- ~/usr/bin/RetroShare  
+   ~/usr/bin/RetroShare  
```

Comment:
~ should be [Install Prefix] instead right?

Reply:
I dont know we use same text like years, i only edited it the space